catroid/src/main/java/org/catrobat/catroid/formulaeditor/FormulaElement.java

```diff
@@ -574,6 +576,18 @@ private Object interpretFunctionContains(Object right, Sprite sprite) {
 		return 0d;
 	}
 
+	private Object interpretFunctionIndexOfItem(Object left, Sprite sprite) {
+		if (rightChild.getElementType() == ElementType.USER_LIST) {
+			Project currentProject = ProjectManager.getInstance().getCurrentProject();
```

Comment:
pls ask @DinosaurierRex possible to avoid ProjectManager.getInstance().getCurrentProject()

Reply:
I will leave that as it is for now, because there is no dependency injection for the project manager implemented yet